.huskyrc.json

```diff
@@ -0,0 +1,9 @@
+{
+    "husky": {
+      "hooks": {
+        "pre-commit": "lint-staged",
+        "commit-msg": "commitlint -E HUSKY_GIT_PARAMS",
+        "prepare-commit-msg": "exec < /dev/tty && git cz --hook || true"
+               }
+            }
```

Comment:
fix json syntax

Reply:
It's fixed in the other PR. I'll merge it onto this one.